nbs/docs/tutorials/20_conformal_prediction.ipynb

```diff
@@ -0,0 +1,230 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Conformal Prediction\n",
+    "> Tutorial on how to train neuralforecast models and obtain prediction intervals using the conformal prediction methods"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Conformal prediction intervals use cross-validation on a point forecaster model to generate the intervals. This means that no prior probabilities are needed, and the output is well-calibrated. No additional training is needed, and the model is treated as a black box. The approach is compatible with any model.\n",
+    "\n",
+    "In this notebook, we demonstrate how to obtain intervals using the conformal prediction methods."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "import os\n",
+    "import tempfile\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "from neuralforecast import NeuralForecast\n",
+    "from neuralforecast.models import NHITS\n",
+    "from utilsforecast.losses import mae, rmse, smape\n",
+    "from neuralforecast.utils import AirPassengersPanel, AirPassengersStatic"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logging.getLogger('pytorch_lightning').setLevel(logging.ERROR)\n",
+    "os.environ['NIXTLA_ID_AS_COL'] = '1'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data\n",
+    "\n",
+    "We simply use AirPassengers dataset for the demostration of conformal predictions.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "AirPassengersPanel_train = AirPassengersPanel[AirPassengersPanel['ds'] < AirPassengersPanel['ds'].values[-12]].reset_index(drop=True)\n",
+    "AirPassengersPanel_test = AirPassengersPanel[AirPassengersPanel['ds'] >= AirPassengersPanel['ds'].values[-12]].reset_index(drop=True)\n",
+    "AirPassengersPanel_test['y'] = np.nan\n",
+    "AirPassengersPanel_test['y_[lag12]'] = np.nan"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also create this directory structure with a spark dataframe using the following:"
```

Comment:
? Is the Spark part necessary here? I think for most users it's easier to just show on a Pandas df